### ISSUE TITLE
Meetings that are being modifed are being set to Unpublished

### ISSUE THREAD

**Issue by paulnagle** [bmlt-enabled/bmlt-workflow#163]
Hello,
   The last couple of meetings that have gone through the workflow, have ended up being set to Unpublished.

The only changes that were made were changes to the time or location of the meetings, yet when I check in BMLT after approving the changes in the the workflow, the meetings are Unpublished.

Maybe you need to set published to 1 explicitly in https://github.com/bmlt-enabled/bmlt-workflow/blob/main/src/REST/Handlers/SubmissionsHandler.php#L401
```php
$change['published'] = 1;
```

**Comment by nigel-bmlt** [comment]
Could you confirm which version of bmlt you are attaching to?

**Comment by paulnagle** [comment]
BMLT Root Server Version 3.0.1

**Comment by paulnagle** [comment]
Actually, I am running BMLT version 3.0.3. I upgraded on Feb 24th, and have been seeing the issue since. 
The workflow plugin is reporting it as 3.0.1 though
<img width="711" alt="image" src="https://user-images.githubusercontent.com/3714999/226113128-8b2e2b2f-daf2-4894-b761-a5270d3a6194.png">
I have the workflow Plugin Version 1.1.2 installed.

**Comment by jbraswell** [comment]
Are you actively able to reproduce this issue? If you submit a change to any meeting, it always unpublished it?

**Comment by paulnagle** [comment]
Yes. I just tried again by slightly changing the name of the group through the workflow plugin, and the meeting became unpublished.

**Comment by jbraswell** [comment]
@nigel-bmlt can you get me the exact request body and http verb that's being sent to the root server rest api?

**Comment by paulnagle** [comment]
[bmlt_workflow_debug.txt](https://github.com/bmlt-enabled/bmlt-workflow/files/11008885/bmlt_workflow_debug.txt)
Debug log file attached...
